test/common/pool_trace.h

```diff
@@ -1,9 +1,9 @@
-// Copyright (C) 2023 Intel Corporation
+// Copyright (C) 2023-2024 Intel Corporation
 // Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-#ifndef UMF_TRACE_POOL_H
-#define UMF_TRACE_POOL_H
+#ifndef UMF_TEST_TRACE_POOL_H
```

Comment:
pool_trace

If we want to have an include guard based on the file name, fix it here and elsewhere